tests/default/controllers/test_suite_supervisor/test_suite_supervisor.py

```diff
@@ -240,6 +240,7 @@ def run(self):
 
         # 30 seconds before executing the next world, 60 seconds for the robot_window_html test
         delay = 60 if self.currentSimulationFilename.endswith('/robot_window_html.wbt') else 30
+        delay = delay * 10
```

Comment:
There's no need to increase the delay during the linux tests, and doing so could make it take longer to find and debug test failures.
```suggestion
        if sys.platform == "darwin":
            # Longer timeout on MacOS because Webots takes longer to start there during CI.
            delay = delay * 10
```